GraphicsView/include/CGAL/Qt/camera.h

```diff
@@ -19,6 +19,7 @@
  WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
 
 *****************************************************************************/
+// SPDX-License-Identifier: GPL-3.0+
```

Comment:
The license notice says:
> This file may be used under the terms of the GNU General Public License 
 versions 2.0 or 3.0 as published by the Free Software Foundation

That cannot be GPL-3.0+. We can use it under GPL-3.0 (with the `+`).